metrics/service.go

```diff
@@ -70,22 +72,30 @@ func (s *service) ProxyHandlerMiddleware(next func(c *proxy.Context)) func(c *pr
 	return func(c *proxy.Context) {
 		startTime := time.Now()
 
-		s.proxyNumberOfLiveConnecions.With(prometheus.Labels{
-			"request_type": c.RequestType(),
-		}).Inc()
+		go func() {
+			s.proxyNumberOfLiveConnections.With(prometheus.Labels{
+				"request_type": c.RequestType(),
+				"hostname":     c.WaitHostname(),
+			}).Inc()
+		}()
 
 		next(c)
 
-		s.proxyNumberOfLiveConnecions.With(prometheus.Labels{
+		fmt.Println("DestinationHostNormalized2", c.Hostname())
```

Comment:
i think it was for debug purpose and now we can delete it

Reply:
oh, yeah, good catch.